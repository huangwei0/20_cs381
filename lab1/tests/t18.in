debug off
stopwords off
load b_tracks.txt
preprocess
count tracks
count words
count characters