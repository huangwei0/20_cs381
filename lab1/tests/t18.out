54149
131931
707524
